### PR TITLE
Feat: Service Dashboard Tab

### LINKED ISSUE #790: Service Dashboard Tabs
Figure out what Nav elements to display on the service dashboard. Perhaps a single tab that is always selected.

### PR DESCRIPTION
**Pull request by kaitmore**
Resolves #790

### CODE REVIEW

File: src/components/AppHeader/scenes/UseSDS/components/scenes/Service/index.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { PropTypes } from "prop-types";
+import { connect } from "react-redux";
+import { withRouter } from "react-router-dom";
+
+import Tab from "../../../../../components/Tab";
```
Comment (spmcbride1201) [inline src/components/AppHeader/scenes/UseSDS/components/scenes/Service/index.js]: Can you please refactor these to use module directory imports?

File: src/components/AppHeader/scenes/UseSDS/components/scenes/Service/index.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+            value: instanceCount
+          }
+        ]}
+      />
+    </TabNav>
+  );
+}
+
+function mapStateToProps(state, ownProps) {
+  const { services } = state;
```
Comment (spmcbride1201) [inline src/components/AppHeader/scenes/UseSDS/components/scenes/Service/index.js]: Probably the easiest refactor would be something like this.
`const { fabric: {services} } = state;`

File: src/components/AppHeader/scenes/UseSDS/index.js [hunk 3]
```diff
@@ -2,10 +2,9 @@ import React from "react";
 import { Switch, Route, withRouter } from "react-router-dom";
 
 import FabricAppHeaderContent from "./components/scenes/Fabric";
+import InstanceHeaderContent from "../../components/InstanceHeaderContent";
```
Comment (spmcbride1201) [inline src/components/AppHeader/scenes/UseSDS/index.js]: Please change this to module directory imports style

**Comment by kaitmore** [comment]
@spmcbride1201 I restructured UseSDS & updated mapStatetoProps

**Pull request merged**